public/pages/SampleData/components/SampleDataBox/SampleDataBox.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import ContentPanel from '../../../../components/ContentPanel/ContentPanel';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiSpacer,
+  EuiLink,
+} from '@elastic/eui';
+import { PLUGIN_NAME } from '../../../../utils/constants';
+
+interface SampleDataBoxProps {
+  title: string;
+  icon: any;
+  description: string;
+  loadDataButtonDescription: string;
+  onLoadData(): void;
+  isLoadingData: boolean;
+  isDataLoaded: boolean;
+  detectorId: string;
+}
+
+export const SampleDataBox = (props: SampleDataBoxProps) => {
+  return (
+    <div style={{ height: 'auto' }}>
+      <ContentPanel
+        title={
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              alignContent: 'flexStart',
+            }}
+          >
+            {props.icon}
+            <EuiSpacer size="m" />
+            <h2>&nbsp;&nbsp;</h2>
+            <EuiTitle size="s">
+              <h2 style={{ marginTop: '-3px' }}>{props.title}</h2>
+            </EuiTitle>
+          </div>
+        }
+        titleSize="s"
+        badgeLabel={props.isDataLoaded ? 'INSTALLED' : undefined}
+      >
+        <EuiFlexGroup direction="column" gutterSize="l">
+          <EuiFlexItem grow={false} style={{ height: '70px' }}>
+            <p
+              style={{
+                textAlign: 'justify',
+                lineHeight: 1.4,
+                maxHeight: 4.2,
+              }}
+            >
+              {props.description}
+            </p>
+          </EuiFlexItem>
+          <EuiFlexGroup
+            style={{
+              height: '100px',
+              marginTop: '6px',
+              marginBottom: '0px',
+            }}
+            direction="column"
+            alignItems="center"
+          >
+            <EuiFlexItem grow={false}>
+              <EuiButton
+                style={{ width: '300px' }}
+                data-test-subj="loadDataButton"
+                disabled={props.isLoadingData || props.isDataLoaded}
+                isLoading={props.isLoadingData}
+                onClick={() => {
+                  props.onLoadData();
+                }}
+              >
+                {props.isLoadingData
```

Comment:
Is it possible that sample data loaded into ES index, but fail to create detector? Default maximum detector is 1000. If user have created 1000 detectors, then load sample detector will succeed, but create detector will fail. For this case, will we throw some exception ?